src/styles/blocks/card.scss

```diff
@@ -0,0 +1,92 @@
+.card {
+  width: $cardSize;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
```

Comment:
It would be great to create variables for all colors on the page if you have already started doing this